package.json

```diff
@@ -17,12 +18,15 @@
     "@walletconnect/chat-client": "^0.7.1",
     "@walletconnect/core": "^2.8.0",
     "@walletconnect/push-client": "^0.10.6",
```

Comment:
Might want to do in a separate PR but 0.10.7 is published with the fixed core:

```suggestion
    "@walletconnect/push-client": "^0.10.7",
```

Reply:
Want to do this in the `filter-by-notify` PR since we'll update push client there